code/modules/cargo/packs/gun.dm

```diff
@@ -36,15 +39,15 @@
 	cost = 1000
 	contains = list(/obj/item/gun/ballistic/automatic/pistol/candor)
 
-/datum/supply_pack/gun/revolver
-	name = "Scarborough Revolver Crate"
-	desc = "Contains a concealable Scarborough revolver, chambered in .357."
+/datum/supply_pack/gun/pepperbox
+	name = "HP Firebrand Pepperbox Revolver Crate"
+	desc = "Contains a concealable pepperbox revolver manufactured by the Saint Roumain Militia, chambered in .357."
```

Comment:
Minor terminology gripe here, it’s a “pepperbox pistol,” not a pepperbox revolver. a pepperbox can’t be a revolver, they’re different methods of operation entirely

Reply:
Yknow honestly this isn’t your problem to deal with, ignore it. Maybe if it keeps bugging me I’ll make a pedantic terminologg fix PR